en/docs/authentication-and-authorization/local-development-with-managed-authentication.md

```diff
@@ -0,0 +1,76 @@
+# Develop Web Applications Locally with Choreo’s Managed Authentication
+
+Choreo’s  Managed Authentication capability exposes a set of backend for frontend (BFF) endpoints, facilitating authentication and authorization for web applications. These endpoints are readily available for single-page web applications deployed on Choreo.
+
+As a web application developer, there may be instances where it becomes necessary to run the application on your personal workstation. In such situations, the expectation is for authentication and authorization during local development to work in the same manner as it would when the application is deployed to Choreo, eliminating the need to modify the source code (If you have used OIDC/OAuth2.0 SDKs to integrate authentication and authorization into a web application, you are likely familiar with the common requirement to configure Client Secrets locally for enabling logins and API calls when running the application locally).
```

Comment:
@Nashaath shall we remove the part within brackets. We decided to remove that comparison as it seems unnecessary here

Reply:
Done.